roles/edpm_podman/molecule/disconnected/converge.yml

```diff
@@ -0,0 +1,57 @@
+---
+# Copyright 2019 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  tasks:
+    - name: Run install
+      include_role:
+        name: osp.edpm.edpm_podman
+        tasks_from: install.yml
+        vars_from: "redhat.yml"
+      vars:
+        edpm_podman_disconnected_ocp: true
+        edpm_podman_registries_conf: |
+           unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
+           short-name-mode = ""
+
+           [[registry]]
+             prefix = ""
+             location = "quay.io/okd/scos-content"
+
+             [[registry.mirror]]
+               location = "quay.io/molecule/okd"
+               pull-from-mirror = "digest-only"
+
+           [[registry]]
+             prefix = ""
+             location = "quay.io/okd/scos-release"
+
+    # https://bugs.launchpad.net/bugs/1889510
+    - name: Test podman network ls
+      become: true
+      command: podman network ls
+
+    - name: Assert registries.conf matches
+      ansible.builtin.lineinfile:
+        name: /etc/containers/registries.conf
+        search_string: 'location = "quay.io/okd/scos-content"'
+        line: 'changed'
+        state: present
+      check_mode: true
+      register: registries
+      failed_when: registries is not changed
```

Comment:
Is this testing anything at all? `failed_when: registries is not changed` means it would pass when `changed` is written in place of ` location = "quay.io/okd/scos-content"` as mentioned in `line`  and `search_string`.